packages/data-schema/src/ModelType.ts

```diff
@@ -29,6 +29,9 @@ import type { methodKeyOf } from './util/usedMethods.js';
 const brandName = 'modelType';
 export type deferredRefResolvingPrefix = 'deferredRefResolving:';
 
+type BaseAllowModifier = Omit<AllowModifier, 'resource'>;
+type AnyAuthorization = Authorization<any, any, any>;
```

Comment:
Since these type aliases are related to Auth specifically, let's move them into `Authorization.ts` and import them into this file. 

In a follow up PR, we can also do a global find/replace of `Authorization<any, any, any>` -> `AnyAuthorization`. Should be many cases throughout the code base.